mass_mailing_custom_unsubscribe_event/demo/assets.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Tecnativa - João Marques
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<template id="assets_frontend_demo"
+          inherit_id="web.assets_frontend">
```

Comment:
```suggestion
<template id="assets_tests"
          inherit_id="web.assets_tests">
```

See https://github.com/odoo/odoo/blob/c92f058571b0656451f24e24154dd1bdb3bdb48a/addons/web/views/webclient_templates.xml#L407 (new in v13)